multi-int-store/src/main/java/com/connexta/multiintstore/common/adaptors/S3StorageAdaptor.java

```diff
@@ -50,9 +48,14 @@ public void store(
             .build();
     final RequestBody requestBody = RequestBody.fromInputStream(file.getInputStream(), fileSize);
 
-    LOGGER.info("Storing {} in bucket \"{}\" with key \"{}\"", fileName, s3BucketQuarantine, key);
+    log.info("Storing {} in bucket \"{}\" with key \"{}\"", fileName, s3BucketQuarantine, key);
     try {
       s3Client.putObject(putObjectRequest, requestBody);
+      log.info(
+          "Successfully stored \"{}\" in bucket \"{}\" with key \"{}\"",
+          fileName,
+          s3BucketQuarantine,
+          key);
```

Comment:
✏️ Move this out of the try/catch

Reply:
To the end of the method?

Reply:
Yes. We don't want to catch runtime exceptions from Logger#info.

Reply:
🤦‍♂ I should have double checked the exceptions I was catching. Thanks!